modules/specifying-role-arn-albo-sts.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * networking/installing-albo-sts-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="specifying-role-arn-albo-sts_{context}"]
+= Specifying the role ARN for ALBO on an STS cluster
+
+The role ARN needs to be passed to the Operator as an environment variable. You can use the dedicated input box in the OperatorHub web UI or specify it in the Subscription resource when installing the Operator by using the OpenShift CLI.
```

Comment:
should you define ARN before this acronym? Or is it generally well-known? Amazon Resource Name

Reply:
Good point, yes, I think it should be defined.